packages/node-integration-tests/README.md

```diff
@@ -1,6 +1,15 @@
-# Integration and performance tests for nodejs
+# Integration Tests for @segment/analytics-node
+
+## Tests
+| Test Path | Description |
+| ---- | ----------- |
+| [./src/durability-tests](src/durability-tests/) | Test that all the events created by the Analytics SDK end up as HTTP Requests, and that graceful shutdown does not result in events getting lost. |
```

Comment:
How do we run the durability tests? I only see perf tests in the package.json.

Reply:
oops, I did not push everything yet